var/spack/repos/builtin/packages/rocm-openmp-extras/package.py

```diff
@@ -113,6 +115,11 @@ class RocmOpenmpExtras(Package):
         depends_on('hsa-rocr-dev@' + ver, when='@' + ver)
         depends_on('llvm-amdgpu@{0} ~openmp'.format(ver),
                    when='@' + ver)
+    if ver == 'master':
```

Comment:
Indentation seems off here. I believe you can get rid of the if condition and use when=@master instead.

Reply:
Done